bigdecimal.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
   s.summary       = "Arbitrary-precision decimal floating-point number library."
   s.description   = "This library provides arbitrary-precision decimal floating-point number class."
   s.homepage      = "https://github.com/ruby/bigdecimal"
-  s.license       = "Ruby"
+  s.license       = "bsd-2-clause"
```

Comment:
```suggestion
  s.license       = ["Ruby", "bsd-2-clause"]
```

Reply:
I don't think ruby is a license

Ruby the language uses bsd-2-clause as it's license
https://www.ruby-lang.org/en/about/license.txt